### ISSUE TITLE
Plan the WCAG 2.2 Guidance Design Sprint

### ISSUE THREAD

**Issue by CharlotteDowns** [alphagov/govuk-design-system#2814]
## What

All the planning activities for the WCAG 2.2 Guidance Design Sprint 3 July 2023. This relates to the epic [WCAG 2.2 AA updates to guidance to comply, and help teams to comply, with new criteria #2729](https://github.com/alphagov/govuk-design-system/issues/2729).

## Why

We want to determine what type of guidance we need and how it will look in the Design System for when WCAG 2.2 is published later this year.

## Who needs to work on this

Interaction Designer, Content Designer 

## Who needs to review this

The Design Sprint participant team (Accessibility specialist, Content Designer, Designer, Developer, Technical writer)

## Useful links

[Design Sprint for WCAG 2.2 Content Updates](https://docs.google.com/document/d/1nSp_EZW3BBZyASXgg05sla8N-WQa-GD3Hmdyf0tE0qM/edit#
)

## Done when

**This sprint**
- [x] Contacted potential speakers
- [x] Define scope and activities on Day 1
- [x] Confirm list of participants (roughly 7 participants)
- [x] Confirm all participants have been excluded from support/stand up facilitation
- [x] Book any in-person rooms ([The White Chapel Building](https://www.google.com/maps/place/The+White+Chapel+Building/data=!4m2!3m1!19sChIJAQDA0EoDdkgRcKO8L2VV908)??)
- [x] Post user research question on Slack and Friday x-gov chat
- [x] Look at search box analytics and content audit user research for related user research
- [x] Determine tools we are going to use

**To get out of Blocked**
- [x] Confirm new date
- [x] Rebook rooms
- [x] Reconfirm with speakers
- [x] Go through deck and adjust timings to day by day schedules

**Next sprint**
- [x] Prepare any boards for any online tools we are using
- [x] Send out joining information and guidance on how we are going to use tools
- [x] Collect in-person materials: post-its, pens etc.
- [x] Prepare [deck from template](https://docs.google.com/presentation/d/1HTf3-MiEm-OHtvbvgviCy7Qi6k9p-yBrMdhfhlcn5j0/edit#slide=id.g2f64e8e1cd_0_287) with timings/activities

Not sure whether to do this
- [ ] On the work done so far, highlight which content is out-of-scope

## Next work
What we are doing next can be found in [Next steps for WCAG 2.2 Guidance Design Sprint #2971](https://github.com/alphagov/govuk-design-system/issues/2971)

**Comment by CharlotteDowns** [comment]
BLOCKED till we can find a new date to run it.

**Comment by CharlotteDowns** [comment]
Links

[Our slide deck](https://docs.google.com/presentation/d/1zWJJwbZdGWMfVBM2gv7vK616-LDaRGy9GoJHqvawgTs/edit?pli=1#slide=id.g24ef1679a14_0_659).


- [Responses from our users on Slack and Slido](https://docs.google.com/document/d/1gZOT-U1N2zgUTOvmTXfon_lKYJYNMzRCuwDFdHgVIkE/edit?usp=sharing)
- [Search box analysis for Accessibility](https://docs.google.com/spreadsheets/d/1sYyu1dd5A2uvXj3rZnsq2Aosm__8l1slEO0WUOzkFr4/edit?pli=1#gid=148966734)
- [How might we? exercise padlet](https://padlet.com/charlottedowns/how-might-we-hmw-8hiekgi84we8ujjg)
- [Map it out exercise padlet](https://padlet.com/charlottedowns/map-it-out-7y94b06d2do508fn)

**Comment by CharlotteDowns** [comment]
## Prototype outcomes

- [Group C (Community support sessions prototype) drafted page content.](https://docs.google.com/document/d/1R3Rl5Jw8DnJxbxR789vlNOqPaDoJcexdeXBWpEP-_eg/edit#heading=h.ob8gvahifrkr)
- [Group Z (Website changes) pull request.](https://github.com/alphagov/govuk-design-system/pull/2955)
